internal/config/config.go

```diff
@@ -34,10 +34,11 @@ import (
 )
 
 const (
-	DefaultNamespace             = "default"
-	VulnerabilityType            = "vuln_mgmt"
-	AssetInventoryType           = "asset_inventory"
-	ResultsDatastreamIndexPrefix = "logs-cloud_security_posture.findings"
+	DefaultNamespace                = "default"
+	VulnerabilityType               = "vuln_mgmt"
+	AssetInventoryType              = "asset_inventory"
+	defaultFindingsIndexPrefix      = "logs-cloud_security_posture.findings"
```

Comment:
Why is this called default it doesn't mention any namespace?

Reply:
This is why it's a prefix. The whole index is constructed as `{prefix}-{namespace}`. The prefix can vary - see the prefix for Vulnerability events.

Reply:
Here's an example what is being sent for CNVM

```yaml
  - id: cloudbeat/vuln_mgmt_aws-vuln_mgmt-6921c0f7-ae37-4a1b-910b-f158a7c1af64
    name: cnvm
    revision: 1
    type: cloudbeat/vuln_mgmt_aws
    use_output: default
    meta:
      package:
        name: cloud_security_posture
        version: 1.9.0
    data_stream:
      namespace: default
    package_policy_id: 6921c0f7-ae37-4a1b-910b-f158a7c1af64
    streams:
      - id: >-
          cloudbeat/vuln_mgmt_aws-cloud_security_posture.vulnerabilities-6921c0f7-ae37-4a1b-910b-f158a7c1af64
        data_stream:
          dataset: cloud_security_posture.vulnerabilities
          type: logs
        period: 24h
```

So the datastream is built by `<type>-<dataset>-<namespace>`